pytorch_lightning/trainer/optimizers.py

```diff
@@ -119,21 +119,8 @@ def configure_schedulers(
         lr_schedulers = []
         default_config = _get_default_scheduler_config()
         for scheduler in schedulers:
-            if isinstance(scheduler, dict):
```

Comment:
was the fix to rearrange the if-else tower here? Or was anything else needed to change? Hard to see ^^

Reply:
Nothing has been changed. The exceptions are just being omitted for `manual_optimiation`

Reply:
okay thanks for confirming, I wasn't sure after 10 mins looking at it ^^